backend/api_calls.py

```diff
@@ -877,6 +877,168 @@ def RandomWords(n):
   return '-'.join(words)
 
 
+def SendNotification(request, firebase):
+  """Queue a notification to be sent.
+
+  Validation:
+    notificationId does not exist.
+    groupId or playerId exists.
+
+  Args:
+    notificationId: Queued Notification id to create.
+    message: Message to send to client.
+    previewMessage: Short preview of the message conents. Optional.
```

Comment:
All the other non-update methods don't have any optional arg. Could we require this arg be present even if not set?